.github/workflows/contrib-tests.yml

```diff
@@ -45,12 +45,17 @@ jobs:
           pip install .[retrievechat-qdrant]
       - name: Install unstructured when python-version is 3.9 and on linux
         run: |
+          if [[ ${{matrix.os}} != ubuntu-latest ]]; then
```

Comment:
Why not use `runs-on: ubuntu-latest`

Reply:
it disrupts the intended functionality

Reply:
Feel like we might want to separate the actions for different contrib tests. Changes made to one contrib class shouldn't need to trigger all contrib tests. @sonichi

Reply:
yeah, this should work! @ekzhu